### DISCUSSION TITLE
Slovenian sentences that should be supported

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#398] [Supported Sentences]
If you don't feel comfortable contributing to the YAML in this repository, you can post your **Slovenian** (`sl`) sentences that match [our supported intents](https://github.com/home-assistant/intents/blob/main/intents.yaml) below.

**Comment by andrejs2** [comment]
Hello to everyone, including Slovenian developers who might be able to assist further.

I have very limited coding knowledge (it's not my profession at all), but I'm deeply passionate about IoT and home automation. After spending several days translating and attempting to make the right sentences and intents work, I find myself stuck with only one successful test (`light_HassTurnOff.yaml`). The rest of the sentences are derived from Croatian sentences with the assistance of @spuljko and @tetele.

If there are any Slovene-speaking individuals willing to lend a hand or even review the code in [my fork](https://github.com/andrejs2/intents/tree/br_light), I would greatly appreciate it. I'm unsure where to proceed from here, and every hint or suggestion would be invaluable. Not to mention slovenian language (grammar) is quite challanging :)

Where am I encountering issues? Most of the sentences are triggering errors due to incorrect responses.

`FAILED tests/test_language_sentences.py::test_light_HassTurnOn[sl] - AssertionError: Incorrect response for: Vključi luči v kuhinji`


Edit:

Sentence fixed.
